test/modules/test_modules_remind.py

```diff
@@ -28,23 +28,35 @@ def sopel():
     ('13:37 message', TimeReminder('13', '37', None, None, 'message')),
     ('13:37:00 message', TimeReminder('13', '37', '00', None, 'message')),
     # Make sure numbers are not confused with anything else
-    ('13:37:00 10 message',
-     TimeReminder('13', '37', '00', None, '10 message')),
+    (
```

Comment:
You'll hate me so hard if #1590 get merged first, and I won't blame you for that. I mean, I already hate myself a little bit for that...

Reply:
:stuck_out_tongue: